lib/report_formatter/timeline.rb

```diff
@@ -106,12 +106,21 @@ def tl_event(tl_xml, row, col)
             ems = ExtManagementSystem.find(rec[:ems_id])
             ems_cloud =  true if ems.kind_of?(EmsCloud)
           end
-          if rec[:vm_name] && !ems_cloud             # Create the title using VM name
-            e_title = rec[:vm_name]
-          elsif rec[:host_name] && !ems_cloud                 #   or Host Name
-            e_title = rec[:host_name]
-          elsif rec[:ems_cluster_name] && !ems_cloud          #   or Cluster Name
-            e_title = rec[:ems_cluster_name]
+          if !ems_cloud
```

Comment:
> @dkorn I expect to have a:
> 
> when "Container"
> what does it take for that? (I assume we're missing something if you're not doing it)

IMHO, a generic case expression isn't the right solution here (if that's what you meant).

first, there is no one specific variable or attribute we are checking here. I assume a hack can be written but to my understanding that isn't the "classic" use case.

second, and in a way related, it doesn't suit the logic I'm trying to follow here. the event title should be the one of the most inner entity it relates to. for example, in case both ```rec[:container_name]``` and ```rec[:container_group_name]``` have values the event title should be the *container* name and not the pod's.